traffic_ops/traffic_ops_golang/api/shared_handlers.go

```diff
@@ -25,15 +25,13 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
-	"github.com/apache/trafficcontrol/lib/go-rfc"
+	"github.com/apache/trafficcontrol/lib/go-log"
+	"github.com/apache/trafficcontrol/lib/go-tc"
```

Comment:
Shouldn't move these up into the standard imports, they should stay beneath in a separate set of lines.